### PR TITLE
to_csv & to_dataframe

### PR DESCRIPTION
**Pull request by oleurud**

### CODE REVIEW

File: cartoframes/data/observatory/catalog/dataset.py [hunk 1]
```diff
@@ -365,6 +365,60 @@ def get_all(cls, filters=None, credentials=None):
 
         return cls._entity_repo.get_all(filters, credentials)
 
+    @check_do_enabled
```
Comment (Jesus89) [inline cartoframes/data/observatory/catalog/dataset.py]: A minor question. Why moving this to the `classmethod` section?

**Comment by alasarr** [comment]
@Jesus89 not sure about this. BigQuery for example call it (to_dataframe)[https://googleapis.dev/python/bigquery/latest/generated/google.cloud.bigquery.job.QueryJob.html#google.cloud.bigquery.job.QueryJob.to_dataframe] and for me it was pretty obvious. I'd keep as it's since is less verbose and it's easy to explain at the doc

File: cartoframes/data/observatory/catalog/dataset.py (hunk 1 above)
Comment (oleurud) [reply cartoframes/data/observatory/catalog/dataset.py]: Because it was in the download one too. Not sure what is your doubt

**Comment by Jesus89** [comment]
OK, it's true that it's clear enough with the doc (and or course less verbose) :+1:

File: cartoframes/data/observatory/catalog/dataset.py (hunk 1 above)
Comment (Jesus89) [reply cartoframes/data/observatory/catalog/dataset.py]: Usually, we keep the structure `classmethod`s, `staticmethod`s, `instance methods`, and now it's mixed, but it's a minor thing, just asking if there was a reason.

**Comment by Jesus89** [comment]
:+1:

**Pull request merged**